python/pyspark/pandas/tests/test_series.py

```diff
@@ -2987,9 +2987,9 @@ def test_argmin_argmax(self):
             name="Koalas",
         )
         psser = ps.from_pandas(pser)
-
         self.assert_eq(pser.argmin(), psser.argmin())
         self.assert_eq(pser.argmax(), psser.argmax())
+        self.assert_eq(pser.argmax(skipna=False), psser.argmax(skipna=False))
```

Comment:
Can we have one more test for chained operation while we're here ?

e.g.
```python
(pser + 1).argmax(skipna=False)
```

Reply:
Certainly!